spec/requests/search_spec.rb

```diff
@@ -1,64 +1,70 @@
 # frozen_string_literal: true
-describe SearchController, type: :controller do
+
+describe 'Search', type: :request do
   let(:user) { create(:user) }
-  let(:user2) {  create(:user, email: 'bar@email.com') }
-  let(:user3) {  create(:user, email: 'foo@email.com', banned: true) }
+  let!(:user2) do
+    create(:user, email: 'bar@email.com',
+                  name: 'user 2 name')
+  end
+  let!(:user3) do
+    create(:user, email: 'foo@email.com',
+                  banned: true,
+                  name: 'user 3 name')
+  end
 
   describe '#index' do
     context 'when user is logged in' do
-      include_context :logged_in_user
+      before { sign_in user }
 
       context 'when have passed email' do
         it 'strip passed string' do
-          expect_any_instance_of(String).to receive(:strip)
-          get :index, params: { search: { email: '  hi@email.com ' } }
+          get search_index_path, params: { search: { email: 'bar@email.com' } }
           expect(response).to render_template(:index)
```

Comment:
Forgot to mention, it is [recommended to avoid the rails-controller-testing gem](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/#comment-3869319771) so if you assert that going to this URL is successful, that should be sufficient 👍 

`render_template` is only supported by the gem I mentioned above.

Reply:
got it. Will do this change.